README.md

```diff
@@ -120,65 +123,82 @@ exec { 'test':
+
 Sets additional environment variables to set for a command. Valid options: String, or an array of multiple options. Default: Undefined.
 
 ##### `logoutput`
-Defines whether to log command output in addition to logging the exit code. If you specify 'on_failure', it only logs the output when the command has an exit code that does not match any value specified by the `returns` attribute. Valid options: 'true', 'false', and 'on_failure'. Default: 'on_failure'.
+
+Defines whether to log command output in addition to logging the exit code. If you specify 'on_failure', it only logs the output when the command has an exit code that does not match any value specified by the `returns` attribute. Valid options: true, false, and 'on_failure'. Default: 'on_failure'.
 
 ##### `onlyif`
+
 Runs the exec only if the command returns 0. Valid options: String. Default: Undefined.
 
 ##### `path`
+
 Specifies the search path used for command execution. Valid options: String of the path, an array, or a semicolon-separated list. Default: Undefined.
 
 ##### `refresh`
+
 Refreshes the command. Valid options: String. Default: Undefined.
 
 ##### `refreshonly`
-Refreshes the command only when a dependent object is changed. Used with `subscribe` and `notify` [metaparameters](http://docs.puppet.com/references/latest/metaparameter.html). Valid options: 'true', 'false'. Default: 'false'.
+
+Refreshes the command only when a dependent object is changed. Used with `subscribe` and `notify` [metaparameters](http://docs.puppet.com/references/latest/metaparameter.html). Valid options: true, false. Default: false.
 
 ##### `returns`
-Lists the expected return code(s). An error will be returned if the executed command returns something else. Valid options: An array of acceptable return codes or a single value. Default: 0.
+
+Lists the expected return code(s). If the executed command returns something else, an error is returned. Valid options: An array of acceptable return codes or a single value. Default: 0.
 
 ##### `timeout`
+
 Sets the maximum time in seconds that the command should take. Valid options: Number or string representation of a number. Default: 300.
 
 ##### `tries`
+
 Determines the number of times execution of the command should be attempted. Valid options: Number or a string representation of a number. Default: '1'.
 
 ##### `try_sleep`
+
 Specifies the time to sleep in seconds between `tries`. Valid options: Number or a string representation of a number. Default: Undefined.
 
 ##### `unless`
+
 Runs the `exec`, unless the command returns 0. Valid options: String. Default: Undefined.
 
 ## Limitations
 
- * Only supported on Windows Server 2008 and above, and Windows 7 and above.
+* Only supported on Windows Server 2008 and above, and Windows 7 and above.
+
+* Only supported on Powershell 2.0 and above.
 
- * Only supported on Powershell 2.0 and above.
+* To use here-strings in scripts for this module, you must use the `@" / "@` syntax in your code.
```

Comment:
When using here-strings within inline or templated scripts executed by this module, you must use the double-quote style syntax that begins with `@"` and ends with `"@`.  The single-quote syntax that begins with `@'` and ends with `'@` is not supported.

Note that any external `.ps1` script file loaded or executed with the call operator `&` is not subject to this limitation, and may contain any style here-string. For instance, the script file `external-code.ps1` may contain any style here-string:

~~~ puppet
exec { 'external-code':
  command   => '& C:\external-code.ps1',
  provider  => powershell,
}
~~~